src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/entity/ActionEntity.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.sentenceplan.entity
+
+import jakarta.persistence.Convert
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.springframework.data.jpa.repository.JpaRepository
+import java.time.ZonedDateTime
+import java.util.UUID
+
+@Entity(name = "Action")
+@Table(name = "action")
+class ActionEntity(
+
+  val objectiveId: UUID,
+  var description: String,
+  @Convert(converter = NumericToBooleanConverter::class)
```

Comment:
I think you can use Hibernate for this - "boolean" converts to true/false to 1/0
```suggestion
  @Type(type = "boolean")
```